db/migrations/1564271714995_create-key-parameter-and-network-entity-reports.js

```diff
@@ -789,13 +789,238 @@ FROM motorola_cm."cell_x_export" t1
 
 `;
 
+const NETWORK_RELATIONS = `
+--HUAWEI 3G3G RELATIONS 
+select 
+'Huawei' as "SRV  VENDOR",
+'3G' as "SRV TECH",
+hex_to_int(REPLACE(t3.data->>'LAC','H''','')) as "SRV LAC",
+t1.data->>'CELLID' as "SRV  CELLID",
+t3.data->>'CELLNAME' as "SRV  CELLNAME",
+'3G' as "NBR TECH",
+hex_to_int(REPLACE(t2.data->>'LAC','H''','')) as "NBR LAC", 
+(t1.data->>'NCELLID')::INTEGER as "NBR CELLID",
+t2.data->>'CELLNAME' as "NBR CELLNAME"
+from huawei_cm."UINTRAFREQNCELL" t1
+inner join huawei_cm."UCELL" t2 on t1.data->>'FILENAME'=t2.data->>'FILENAME' and t1.data->>'NCELLID'= t2.data->>'CELLID' and t1.data->>'RNCID' = t1.data->>'NCELLRNCID'
+inner join huawei_cm."UCELL" t3 on t1.data->>'FILENAME'=t3.data->>'FILENAME' and t1.data->>'CELLID'= t3.data->>'CELLID'
+union
+--HUAWEI 3G3G EXT RELATIONS
+select
+'Huawei' as "SRV  VENDOR",
+'3G' as "SRV TECH",
+hex_to_int(REPLACE(t5.data->>'LAC','H''','')) as "SRV LAC",
+t3.data->>'CELLID'as "SRV  CELLID",
+t5.data->>'CELLNAME' as "SRV  Cell Name",
+'3G' as "NBR TECH",
+hex_to_int(REPLACE(t4.data->>'LAC','H''','')) as "NBR LAC",
+(t3.data->>'NCELLID')::INTEGER as "NBR CELLID" ,
+t4.data->>'CELLNAME' as "NBR Cellname"
+from huawei_cm."UINTRAFREQNCELL" t3
+inner join huawei_cm."UEXT3GCELL" t4 on t1.data->>'FILENAME'=t2.data->>'FILENAME' and t4.data->>'NCELLID' = t4.data->>'CELLID' and t3.data->>'RNCID' <> t4.data->>'NRNCID'
+inner join huawei_cm."UCELL" t5 on t1.data->>'FILENAME'=t5.data->>'FILENAME' and t3.data->>'CELLID'= t5.data->>'CELLID'
```

Comment:
This query seems to have an error. It is referencing **t1.data** . It does not exist.
```
inner join huawei_cm."UCELL" t5 on **t1**.data->>'FILENAME'=t5.data->>'FILENAME' and t3.data->>'CELLID'= t5.data->>'CELLID'
```

Reply:
corrected